test/integration/capacity.test.ts

```diff
@@ -90,8 +90,17 @@ describe('capacity', () => {
       expect(capacity.latestTokenId).to.equal(mockTokenId)
       expect(capacity.originalTokenId).to.equal(mockTokenId)
     })
+
+    it('it should get transactions', async () => {
+      identitySelfMock()
```

Comment:
shouldn't be required as requesting a transaction doesn't hit the identity service

Reply:
this is for phuture

Reply:
what?

Reply:
what is phuture?